src/forms/FormFieldImageVideoLinkDocumentBrowsing/ImageVideoLinkDocumentBrowsing.styled.tsx

```diff
@@ -53,8 +53,11 @@ export const BrowseSpan = styled.span`
   color: ${theme.colors.gray600};
   font-family: ${theme.fontFamily};
   font-size: 16px;
-  margin-bottom: 24px;
-  margin-top: 43px;
+
+  &.has-options {

```

Comment:
Same comment as in the main file

Reply:
In BrowseOptionsContainer (line 17) we no longer need the margin bottom since the parent component has a style of display: flex and justify-content: center. The only thing we should add there is the margin space between the options and the label, which I believe should be 24px.